### ISSUE TITLE
NoMethodError: undefined method 'password_length' for #<Class:0x000055c3f5335fe0>

### ISSUE THREAD

**Issue by davidwessman** [scambra/devise_invitable#851]
Hello!

Versions:

- devise 4.7.3
- devise_invitable 2.0.4

```ruby
inviter = User.first
User.invite!({email: "email@test.com"}, inviter) 
=> NoMethodError: undefined method 'password_length' for #<Class:0x000055c3f5335fe0>
```

I notice this is only called in the code introduced in 2.0.4 in [lib/devise_invitable/models.rb](https://github.com/scambra/devise_invitable/commit/db4b0b34149f21727f802982a2b12317fa843900).

Is this a bug in the gem or do I need to update how I use it somehow?

```ruby
User.first.password_length => NoMethodError
User.password_length => NoMethodError
Devise.password_length => 6..128
```

**Comment by scambra** [comment]
Same as #850 
do you have validatable in your user model?

**Comment by davidwessman** [comment]
> Same as #850
> do you have validatable in your user model?

No, I do not have `:validatable`

**Comment by scambra** [comment]
Fixed in #850
